spec/support/form_helpers.rb

```diff
@@ -158,6 +159,15 @@ def fill_in_activity_form(
       click_button t("form.button.activity.submit")
     end
 
+    if parent.present? && parent.roda_identifier_fragment.present? && parent.roda_identifier_compound.include?("NF")
```

Comment:
I haven't found a way to use the `is_newton_funded?` method (or any other methods from the model) in this file. I'm also curious to know if there's a way, so if you have any ideas please share :-) 

This line is a bit "customised" because the `fill_in_activity_form` is used by so many specs, I got some failing specs unless I would specify that the identifier fragment was present too

Reply:
Yeah, I've hit up against this problem too. Feel like it's going to bite us every time we add a field that applies to a specific fund

Reply:
It might be clearer to use safe navigation here, i.e. `parent&.roda_identifier_fragment&.include?("NF")`. Still a bit messy, but, as you say, it's tricky to work out the fund in this method

Reply:
@MariluzHerrera The way I proposed to do it (awaiting review) is to create the parent in the spec context, as a Newton-funded programme, then pass that parent as an argument to the form helper method: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/765/files#diff-6b487a8e002822d824047c1d5e7b4bb36ea09be6ecdf0c3e398192bfa920bb76R82

Reply:
@CristinaRO The reason you can't reuse `is_newton_funded?` here, is because this is always `false` for funds. 

Because of this, you can't call `parent.is_newton_funded?` because in the case where the activity being created is a Newton-funded programme (i.e. belongs to a fund whose `roda_identifier_fragment = "NF"`) won't give you the correct result.

I'll fix the implementation however, as `roda_identifier_compound.include?("NF")` would be true for any activity whose Roda ID *contains* `NF` - e.g. GCRF-01-I**NF**O-GB, which shouldn't be the case.

Reply:
@leeky Worth checking / coordinating with @pezholio , who is adding a fund-specific method to identify each fund https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/756/files#diff-b7775c2b256db972e4e9ff2ab4d490a9cab3b71ba82437a2c2ee45f77ab93ad7R337